specification/ai/data-plane/HealthInsights/stable/2024-04-01/openapi.json

```diff
@@ -0,0 +1,3056 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure AI Health Insights",
+    "version": "2024-04-01",
+    "description": "Azure AI Health Insights provides an API that serves insight models, specific for Health & Life Sciences, that perform analysis and provide inferences to be used by a human.",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}/health-insights",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "name": "endpoint",
+        "in": "path",
+        "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).",
+        "required": true,
+        "type": "string",
+        "format": "uri",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "ApiKeyAuth": []
+    }
+  ],
+  "securityDefinitions": {
+    "ApiKeyAuth": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
```

Comment:
Please include a description in the securityDefinition for the generated REST API reference docs. 

See the [TypeSpec docs] for an example of using the @doc decorator to specify a description for the securityDefinition. 

[TypeSpec docs]: https://azure.github.io/typespec-azure/docs/reference/azure-style-guide#examples

Reply:
ok.
Thanks